packages/server/src/helpers/metrics-reporter.js

```diff
@@ -0,0 +1,35 @@
+const COUNTER_METRICS = {
+  db: [
+    { name: 'queries', help: 'DB queries per table' },
+    { name: 'errors', help: 'DB errors' },
+  ],
+}
+
+class MetricsReporter {
+  constructor(app) {
+    this._initializeCounterMetrics(app)
+  }
+
+  dbQuery() {
```

Comment:
Do we actually use this metric at the moment?

Reply:
not yet, I think this was a contrived example just to implement the error/metric handling. All most important metrics currently are HTTP requests by path/code, which is handled by promster. 
I think metrics will be more useful in services so we can reuse this logic there.